apps/tailwind-components/components/Message.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div
+    class="p-3 font-bold flex items-center rounded-input"
+    :class="{
+      'bg-invalid text-required': type == 'invalid',
```

Comment:
The invalid styles do not appear on the preview (Firefox, chrome, etc.).

Reply:
fixed